demos/https/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ afr_module_sources(
     ${AFR_CURRENT_MODULE}
     INTERFACE
         "${CMAKE_CURRENT_LIST_DIR}/iot_demo_https_s3_download_sync.c"
+        "${CMAKE_CURRENT_LIST_DIR}/iot_demo_https_s3_download_async.c"
+        "${CMAKE_CURRENT_LIST_DIR}/iot_demo_https_s3_upload_sync.c"
```

Comment:
upload is not targeted for Aug release. Please keep it separate from this PR

Reply:
Will remove thanks.